setup.py

```diff
@@ -47,6 +47,7 @@
     'chardet<4.0',
     'pymongo>=0.4,<4.0',
     'croniter<0.4',
+    'python-dateutil',
```

Comment:
can you set some version that's tested to work here?

Reply:
@petrjasek done